sonic-net/sonic-buildimage
# table cleanup VNET_MONITOR, BFD_SESSION, ROUTE_TUNNEL

The 3 tables in state DB need to be cleaned up after SWSS restart for have consistat state.


#### Why I did it
Failing to clean up these tables causes an inconsistent state.
##### Work item tracking
- Microsoft ADO **(number only)**:
21695894
#### How I did it
added 3 tables in clean up script.
#### How to verify it
add a vnet route with monitoring. bring up the monitors. restart the swss. readd the route. THe route should not become active.


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
